churchslavonic.tex

```diff
@@ -3,6 +3,9 @@
 
 \def\pkg#1{\textsf{#1}}
 \def\cs#1{\texttt{\textbackslash #1}}
+\providecommand{\XeTeX}{X\kern-.125em\lower.5ex\hbox{Ǝ}\kern-.1667em\TeX}
+\providecommand{\XeLaTeX}{X\kern-.125em\lower.5ex\hbox{Ǝ}\kern-.125em\LaTeX}
+\providecommand{\LuaTeX}{Lua\kern-.125em\TeX}
```

Comment:
Hmm, these are already defined by `\usepackage{xltxtra}`. Are you sure you need these? On my system document compiles fine without these lines...

Commands are defined in a package named `metalogo` that is loaded by `xlxtra`.